packages/data-transport-layer/src/services/main/service.ts

```diff
@@ -79,13 +84,16 @@ export class L1DataTransportService extends BaseService<L1DataTransportServiceOp
     }
 
     await this.state.l1TransportServer.init()
+    this.logger.info('Initialized L1 Transport Server')
```

Comment:
What do you think about adding these loglines inside of the `init` functions?

Reply:
moved these to base-service by logging the name with initialization